doc/ceph-volume/lvm/prepare.rst

```diff
@@ -19,7 +19,7 @@ work with Ceph, the tool will assign a few pieces of metadata information using
 part of a Ceph system, and what role they have (journal, filestore, bluestore,
 etc...)
 
-Although initially :term:`filestore` is supported (and supported by default)
+Although initially :term:`bluestore` is supported (and supported by default)
```

Comment:
@alfredodeza shall we drop "initially"? which objectstore did ceph-volume support first?

Reply:
So yes, this should be updated, probably something like:

```suggestion
Although :term:`bluestore` is the default
```